apps/fluent-tester/src/TestComponents/Menu/MenuFocusZone.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { ButtonV1 as Button } from '@fluentui/react-native';
+import { Menu, MenuItem, MenuTrigger, MenuPopover, MenuList } from '@fluentui-react-native/menu';
+import { Stack } from '@fluentui-react-native/stack';
+
+import { stackStyle } from '../Common/styles';
+
+export const MenuFocusZone: React.FunctionComponent = () => {
```

Comment:
Maybe a different name like "ComponentOutsideMenuList"? The name made me think I'd see a FZ inside the Menu somewhere, so it wasn't obvious what this was testing